admin/scss/bootstrap/_variables.scss

```diff
@@ -403,23 +403,23 @@ $navbar-padding-vertical:           ($spacer / 2);
 
 // Navs
 
-$nav-link-padding:                          12px 8px;
-$nav-link-hover-bg:                         red; //$gray-lighter;
-
-$nav-disabled-link-color:                   $text-muted;
-$nav-disabled-link-hover-color:             $text-muted;
-
-$nav-tabs-border-color:                     #d9dee2;
-
-$nav-tabs-link-border-width:                0;
-$nav-tabs-link-hover-border-color:          green; //$gray-lighter;
-
-$nav-tabs-active-link-hover-bg:             $body-bg;
-$nav-tabs-active-link-hover-color:          $body-color;
-$nav-tabs-active-link-hover-border-color:   #ddd;
-
-$nav-tabs-justified-link-border-color:            #ddd;
-$nav-tabs-justified-active-link-border-color:     $body-bg;
+// $nav-link-padding:                          .5em 1em;
```

Comment:
We are uncommenting stuff as and when we require them and its acting as guide for the default settings. It makes it easier to see what we have customised and what we haven't changed yet. If we delete it all I think that people would create custom code instead of using bootstrap settings.